pl_bolts/datamodules/kitti_datamodule.py

```diff
@@ -21,7 +22,7 @@ class KittiDataModule(LightningDataModule):
 
     def __init__(
             self,
-            data_dir: str,
+            data_dir: Optional[str],
```

Comment:
The default should be `None` here, if it is `Optional`, right? Can you change that?

Reply:
You're absolutely right. Changing it right now!